packages/openapi-spec-types/src/v3/openapi-v3-spec-types.ts

```diff
@@ -0,0 +1,395 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-spec-types
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/*
+ * OpenApiSpec - A typescript representation of OpenApi 3.0.0
+ */
+
+// tslint:disable:max-line-length
+
+import * as OAS3 from 'openapi3-ts';
+// Export spec interfaces from the community module if missing in our package
+export * from 'openapi3-ts';
+
+export type OpenApiSpec = OAS3.OpenAPIObject;
+/**
+ * Custom extensions can use arbitrary type as the value,
+ * e.g. a string, an object or an array.
+ */
+// tslint:disable-next-line:no-any
+export type ExtensionValue = any;
+
+/**
+ * The location of a parameter.
+ * Possible values are "query", "header", "path" or "body".
+ * <p>Specification:
+ * https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md
+ * search "The location of the parameter"
+ */
+export type ParameterLocation = 'query' | 'header' | 'path' | 'cookie';
+
+/**
+ * The style of a parameter.
+ * Describes how the parameter value will be serialized.
+ * (serialization is not implemented yet)
+ * <p>Specification:
+ * https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#responseObject
+ */
+export type ParameterStyle =
```

Comment:
Same comment applies here - can we eventually contribute this to upstream `openapi3-ts` module? Let's leave a comment in this file for now.